src/config.rs

```diff
@@ -0,0 +1,215 @@
+// Copyright The ocicrypt Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Result};
+use std::collections::HashMap;
+
+/// CryptoConfig holding the identifiers and public/private key file data
+/// that will be able to encrypt/decrypt the symmetric key
+#[derive(Debug, Clone)]
```

Comment:
You should be able to derive `Default` as well, and avoid having to define the constructor below.

Reply:
Yes, `Default` can simplify the new() method, I'll use this one.